tests/integrational/ContenttypeTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace tests\integrational;
```

Comment:
I think something like this would suffice: https://github.com/transfer-framework/ezplatform/pull/44/files